proposals/NNNN-Add-new-SDL-System-Structure-using-Mediation-Application.md

```diff
@@ -0,0 +1,119 @@
+
+Basically, in the Mediation Application, the comunicatation with SDL App, SDL Core and SDL HMI is needed. And also, it will be need to absorb the difference between the current system and the new one. Morever, HMI API which HU needs is only send.
+
+Therefore, the changes made in each component to establish this system are shown below.
+
+###### 1. To App Library
+ - Addition of the system identification function
+ - Addition of the function to switch the communication destination and mean
+ - Addition of the function of the communication destination and mean
+
+###### 2. To Mediation Application
+ - Implemantation of SDL Core(using NDK etc.)
+ - Addition of the communication to App Library and notification to SDL Core
+ - Addition of function to determine whether the message from SDL Core is related to new SDL system
+ - Addition of the support function for RPCs except for new SDL system
+ - Addition of the function to determine the display destination(on HU or smartphone)
+ - Addition of the function to manage the conflict of RPC messages
+ - Addition of the screen display function on the smartphone(SDL App list etc.)
+ - Addition of the connection function to SDL App store
+ - Addition of SDL App deletion function
+ - Addition of the function to manage the HMI status
+ - Addition of the function how to reacquire information(such as Policy table etc.) when connecting to another HU
+
+###### 3. To SDL Core
+ - There is no changes from current SDL Core.
+
+###### 4. To SDL HMI
+ - Addition of the guideline for development of HU. (Implementation is handled by each OEM.)
+
+The changes above will be provide a separate proposal for each change after this proposal is accepted.
+
+## Potential downsides
+
+This proposal is to provide new SDL system with low performed ECU such as PTW and low-cost vehicles, thus it assumes that it requires costs of management and maintenance.
+However, it is assumed that the management and maintenance costs will be reduced for the following reasons.
+ - It will use SDL Core same as current one.
+ - There is not any management about SDL HMI, only manages the guideline.
+ - In the case of HU with low performed ECU, there is no OS and the programing language is different. Therefore, generalization is not possible.
+ - The App Library will be shared by adding a function to determine if it is a low performed ECU system.
+From the above, the management and maintenance of the Mediation App would be newly generated.
+
+
+## Impact on existing code
+
+###### 1. App library
+ - As described above, a determination function and a communication destination and method to the new SDL system will be added, but the existing code will not be affected because systems that do not meet the determination conditions or do not have the conditions will continue to operate with the existing basic system.
```

Comment:
```suggestion
 - As described above, a determination function and a communication destination and method to the new SDL system will be added, but the existing code will not be affected because systems that do not meet the determination conditions or do not have the conditions will continue to operate within the existing basic system.
```